Mimir/GraphQL/Types/AvatarType.cs

```diff
@@ -80,7 +99,7 @@ protected override void Configure(IObjectTypeDescriptor<AvatarObject> descriptor
         }
 
         var avatarAddress = context.Parent<AvatarObject>().Address;
-        return (avatarRepo, planetName.Value, avatarAddress);
+        return (avatarRepo, allRuneRepo, planetName.Value, avatarAddress);
```

Comment:
If more repos are added later, it seems we will need to continuously manage this tuple everywhere it is received. Is there a better way to handle this?

Reply:
We're batching some of the elements we need from `AvatarType` into a tuple.
I'm doing this to reduce duplicate code, but I'll have to think a bit more about how to deal with situations like this one, where a change in the shape of the tuple returned by the `GetSource()` function will require a change on the side that uses it.🤔
I can't think of a way off the top of my head, but it's something I'll have to research a bit more.